packages/amplication-nest-logger-module/README.md

```diff
@@ -0,0 +1,15 @@
+# Amplication nestjs logger module
```

Comment:
```suggestion
# Amplication NestJS Logger Module
```
This is a heading, and we mention a technology (NestJS). So we need to capitalize it properly.